components/Nav/NavListItem/NavListItem.tsx

```diff
@@ -3,121 +3,134 @@ import classNames from 'classnames';
-                  >
-                    <span className={styles.linkContent}>{sublink.name}</span>
-                  </a>
-                </Link>
-              </li>
-            ))}
-          </ul>
-        </>
-      )}
-    </li>
-  );
+                    data-testid={`Nav Item ${name}`}
+                >
+                    <span className={styles.linkContent}>{name}</span>
+                    {icon && icon}
+                </a>
+            </Link>
+
+            {hasSublinks && (
+                <>
+                    <button
+                        aria-expanded={areSublinksVisible}
+                        aria-haspopup={hasSublinks}
+                        aria-label="submenu"
+                        className={styles.sublinkToggleButton}
+                        onClick={invertSublinkVisibility}
+                        onMouseEnter={exposeSublinks}
+                        onMouseLeave={hideSublinks}
+                        type="button"
+                    >
+                        {areSublinksVisible ? (
+                            <MinusIcon className={styles.icon} data-testid="minus-icon" />
+                        ) : (
+                            <PlusIcon className={styles.icon} data-testid="plus-icon" />
+                        )}
+                    </button>
+
+                    <ul
+                        className={classNames(styles.sublinksList, {
+                            [styles.invisible]: !areSublinksVisible,
+                        })}
+                        onMouseEnter={exposeSublinks}
+                        onMouseLeave={hideSublinks}
+                    >
+                        {sublinks.map((sublink, index) => (
+                            <li className={styles.sublinkListItem} key={sublink.name}>
+                                {/* 😞 next/link fought being mocked, so `prefetch` has test-specific code */}
+                                {!sublink.isExternal ? (
+                                    <Link href={sublink.href} prefetch={process.env.NODE_ENV === 'production'}>
+                                        <a
+                                            className={styles.link}
+                                            role="link"
+                                            tabIndex={0}
+                                            data-testid={`Nav Item ${sublink.name}`}
+                                            onKeyDown={event => handleKeyDown(event, index)}
+                                        >
+                                            <span className={styles.linkContent}>{sublink.name}</span>
+                                        </a>
+                                    </Link>
+                                ) : (
+                                    <OutboundLink
+                                        analyticsEventLabel={`Clicked on ${sublink.name} -> ${sublink.href}`}
+                                        className={styles.link}
```

Comment:
```suggestion
                                        className={twMerge(styles.link, '[&>svg]:-bottom-2 [&>svg]:right-3')}
```

should finish this work up!

Reply:
Awesome!